packages/jest-util/src/BufferedConsole.ts

```diff
@@ -24,12 +23,13 @@ export default class BufferedConsole extends Console {
     getSourceMaps: () => SourceMaps.SourceMapRegistry | null | undefined,
   ) {
     const buffer: ConsoleType.ConsoleBuffer = [];
-    super(
-      new Writable({
-        write: message =>
-          BufferedConsole.write(buffer, 'log', message, null, getSourceMaps()),
```

Comment:
this should have beena type error - `message` was `Buffer`, not `string`

Reply:
I went with going back to the original implementation anyways, since the stack including lots of internal `readable-stream` frames. I had to pass `6` a `level` which felt really brittle